meinberlin/apps/contrib/templates/meinberlin_contrib/components/card.html

```diff
@@ -0,0 +1,40 @@
+<article class="modul-card" data-add-clickable-area="smart">
+    <header class="card__header">
+        <h3 class="title">This is something for the list</h3>
+        <div class="card__status status">
+            <ul class="pill__list" aria-label="Proposal tags">
+                <li class="pill pill--budget">456.789€</li>
+                <li class="pill pill--category">Category 1</li>
+                <li class="pill pill--info">1 Mehr</li>
+            </ul>
+            <div class="moderator-status" aria-label="Status Indicator">
+                <span
+                    class="moderator-status__color-indicator label--qualified"
+                    aria-hidden="true"
+                ></span
+                ><span
+                    ><strong>Status: </strong>Qualified for next phase</span
+                >
+            </div>
+        </div>
+        <div class="card__meta subtitle text--meta">
+            <p>
+                admin<time datetime="2023-11-06T17:34:26.377384+01:00">
+                    edited am 6. November 2023 - 2023-00023</time
+                >
+            </p>
+        </div>
+    </header>
+    <section class="card__footer--light">
+        <div class="card__stats">
+            <p class="card__stat">0 Support</p>
+            <p class="card__stat">0 Votes</p>
+            <p class="card__stat">0 Comments</p>
+        </div>
+        <a href="/budgeting/2023-00023/" class="more" data-mainlink="true"
+            >Item details<span class="aural"
+                >to: this is something for the list</span
+            ></a
+        >
+    </section>
+</article>
```

Comment:
@hom3mad3 so I am not sure this is the correct way to do a component library now we use more react. Our old library when most of our components were html/django templates was just copied html code with dummy info in a big file and if we updated a component we also had to remember to update the one in the library (see https://github.com/liqd/a4-meinberlin/blob/main/meinberlin/apps/contrib/templates/meinberlin_contrib/component_library.html can be accessed /components) however now we will have alot more react so it would make sense to have a react component library using the actual react components but I don't know how to go about that right now.
Anyway this html version of the card component is now updated to be the same as the react